Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -429,9 +468,10 @@ public double getPrice() {
      * </p>
      *
      * @return {@link String} with ISO 4217 for this currency. Empty string if there is no currency type set
+     * @deprecated Please check {@link BranchEvent} for more info on commerce event tracking with Branch
      */
     public String getCurrencyType() {
-        return currency_.toString();
+        return null;
```

Comment:
if we can return null here, why can't we return null for the above methods `getType()` and `getPrice()`? Sorry if this is a dumb/obvious answer.